joy_teleop/test/test_parameter_failures.py

```diff
@@ -152,36 +152,6 @@ def test_teleop_axis_mappings_missing_axis_or_button(self):
         self.assertEqual(joy_teleop_process.exit_code, 1)
         self.assertTrue('must have an axis, button, or value' in joy_teleop_process.output)
 
-    def test_teleop_axis_mappings_missing_offset(self):
```

Comment:
I think these tests still hold value in showing that things are handled despite some things missing. Is it ok to leave them?

Reply:
Sure thing.